### ISSUE TITLE
NULL arithmetic operations (SQLLOGICTEST #2)

### ISSUE THREAD

**Issue by agershun** [agershun/alasql#198]
Second file ```select2.test``` designed to check, how database works with NULL values. There is a problem, because of difference in how SQL and JavaScript work with null (or undefined) values.

**Comment by agershun** [comment]
I wil add NaN guard to convert NaN to undefined and additional NULL checking:
```js
function und(s,r) {
	return '(y='+s+',typeof y=="undefined"?undefined:'+r+')'
}

function n2u(s) {
	return '(y='+s+',y===y?y:undefined)'
}
```
Plus everywhere in the functions generation we need to add ```var y``` declaration.

**Comment by agershun** [comment]
Fixed iWill be available in version 0.1.7 soon

**Comment by agershun** [comment]
AlaSQL passes 97% SLT tests on NULL operations.
